specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/ResourceProvider.json

```diff
@@ -623,6 +635,10 @@
             "resourceType": {
               "description": "App Service ResourceType meter used for",
               "type": "string"
+            },
+            "osType": {
+              "description": "App Service OS type meter used for",
+              "type": "string"
```

Comment:
what are the valid set of values fo osType, do you think expressing it as x-ms-enum with modelAsString true would be useful?